content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
+
+## Member
+
+Members are *[continuously active]* contributors in the community. They can have issues and PRs assigned to them and tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Kubeflow GitHub organization
+
+### Requirements
+
+- Enabled two-factor authentication on their GitHub account
+- Have made **at least** 2-3 [contributions] to the project or community.
+- Have read the [contributor guide]
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+- **[Open an issue][membership request] against the kubeflow/internal-acls repo**
+  - Ensure your sponsors are @mentioned on the issue
+- **Open a pull request against the kubeflow/internal-acls repo**
+  - Complete every item on the checklist ([preview the current version of the template][membership template])
+  - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship
+- Once your sponsors have responded, your request will be reviewed by the Kubeflow team. Any missing information will be requested
+
+### Responsibilities
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- Subscribed to <kubeflow-discuss@googlegroups.com>
+
+**Note:** members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a primary *reviewer* for the subproject that they are active in.
+
+### Privileges
+
+- Members can do `/lgtm` on open PRs.
+- They can be assigned to issues and PRs, and people can ask members for reviews with a `/cc @username`.
+- Tests can be run against their PRs automatically. No `/ok-to-test` needed.
+- Members can do `/ok-to-test` for PRs that have a `needs-ok-to-test` label, and use commands like `/close` to close PRs as well.
+
+## Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of a subproject. They are knowledgeable about both the codebase and software engineering principles.
+
+**Defined by:** *reviewers* entry in an OWNERS file in a repo owned by the Kubeflow project.
+
+Reviewer status can be scoped to either parts of the codebase or the root directory for the entire codebase
+
+**Note:** Acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file.
+
+- member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
```

Comment:
What about answering user's questions in GitHub issues and Slack ?

Reply:
Added